frontEnd/front-end/src/App.js

```diff
@@ -5,18 +5,21 @@ import 'bootstrap/dist/css/bootstrap.css';
 import AskQuestionPage from './Components/AskQuestionPage';
 import ShowQuestionPage from './Components/ShowQuestionPage'
 import MakeAnswerPage from './Components/MakeAnswerPage';
+import {BrowserRouter, Route, Switch } from  "react-router-dom";
 
 function App() {
   return (
-    <div className="App">
-     <Header /> 
-     <QuestionCard />
-     <QuestionCard />
-     <QuestionCard />
-     <AskQuestionPage/>
-     <ShowQuestionPage/>
-     <MakeAnswerPage/>
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <Header /> 
```

Comment:
You are including your Header here (which is great), but you are still also including this component on the various pages of the app. This means that it is showing twice.

Reply:
Thank you. It has been fixed